cypress/utils/helpers.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const apiRequest = (url, method = 'POST', body = undefined, qs = undefined) =>
+  cy.request({
+    method: method,
+    failOnStatusCode: false,
+    url: url,
+    headers: {
+      'content-type': 'application/json',
+      'osd-xsrf': true,
+    },
+    body: body,
+    qs: qs,
+  });
+
+export const devToolsRequest = (url, method = 'POST', body = undefined, qs = undefined) =>
```

Comment:
nit: this is kind of the wrong name. i think it's technically proxying to the console plugin dev tools is an application that is a child of the console plugin but it is client side.

so we aren't actually firing the request to the ui we are hitting the node server api